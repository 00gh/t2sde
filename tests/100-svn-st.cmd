#!/bin/sh

[ "$QEMU" ] || exit 43

cd $SYSROOT
exe=usr/bin/svn
[ -e $exe ] || exit 42

LANG=C $QEMU -chroot $SYSROOT $exe st usr/src/t2-src
