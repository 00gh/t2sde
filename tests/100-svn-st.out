!       usr/src/t2-src
